msnoise/s002populate_station_table.py

```diff
@@ -40,20 +40,21 @@
 
 MSNoise expects to find
 a file named ``custom.py`` in the current folder. This python file will contain
- a function called ``populate`` wich will accept one argument and return a list
- of stations in the format ``NET_STA``:
+ a function called ``populate`` wich will accept one argument and return a station
```

Comment:
This is a good idea, it does change the API (or at least the "behaviour") somehow. It's OK since next version will be quite different from 1.4, so the API could change and it shouldn't affect too much the users. I've also included the possibility to populate the station table from the data_availability table (current master, see "lazy options" commit), then sync'ing with the dataless will provide the lat/lon details.